File: ios/Classes/OrangeTfliteFlutterHelperPlugin.m
#import "OrangeTfliteFlutterHelperPlugin.h"

@implementation OrangeTfliteFlutterHelperPlugin
+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
  FlutterMethodChannel* channel = [FlutterMethodChannel
      methodChannelWithName:@"orange_tflite_flutter_helper"
            binaryMessenger:[registrar messenger]];
  OrangeTfliteFlutterHelperPlugin* instance = [[OrangeTfliteFlutterHelperPlugin alloc] init];
  [registrar addMethodCallDelegate:instance channel:channel];
}

- (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
  if ([@"getPlatformVersion" isEqualToString:call.method]) {
    result([@"iOS " stringByAppendingString:[[UIDevice currentDevice] systemVersion]]);
  } else {
    result(FlutterMethodNotImplemented);
  }
}

@end

File: ios/Classes/OrangeTfliteFlutterHelperPlugin.swift
import Flutter
import UIKit

public class OrangeTfliteFlutterHelperPlugin: NSObject, FlutterPlugin {
  public static func register(with registrar: FlutterPluginRegistrar) {
    let channel = FlutterMethodChannel(name: "orange_tflite_flutter_helper", binaryMessenger: registrar.messenger())
    let instance = OrangeTfliteFlutterHelperPlugin()
    registrar.addMethodCallDelegate(instance, channel: channel)
  }

  public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
    result("iOS " + UIDevice.current.systemVersion)
  }
}
